### PR TITLE
[Js] Create index with replace flag

### PR DESCRIPTION
**Pull request by eddyxu**
Part of #214

### CODE REVIEW

File: node/package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "tsc": "tsc -b",
     "build": "cargo-cp-artifact --artifact cdylib vectordb-node index.node -- cargo build --message-format=json-render-diagnostics",
     "build-release": "npm run build -- --release",
-    "test": "mocha -recursive dist/test",
+    "test": "npm run tsc; mocha -recursive dist/test",
```
Comment (chebbyChefNEQ) [inline node/package.json:11]: nit: update Readme too.

File: rust/ffi/node/src/index/vector.rs [hunk 2]
```diff
@@ -122,6 +122,14 @@ fn get_index_params_builder(
                 .map_err(|t| t.to_string())?
                 .map(|s| pq_params.max_opq_iters = s.value(cx) as usize);
 
+            obj.get_opt::<JsBoolean, _, _>(cx, "replace")
+                .map_err(|t| t.to_string())?
+                .map(|s| {
+                    println!("Replace is: {:?}", s.value(cx));
```
Comment (chebbyChefNEQ) [inline rust/ffi/node/src/index/vector.rs:128]: use `debug!()`?

**Review by chebbyChefNEQ — APPROVED** [review]

File: node/package.json (hunk 1 above)
Comment (eddyxu) [reply node/package.json:11]: fixed.

File: rust/ffi/node/src/index/vector.rs (hunk 2 above)
Comment (eddyxu) [reply rust/ffi/node/src/index/vector.rs]: deleted

**Pull request merged**